azure-pipelines/end-to-end-tests-dir/native-platform-dep.ps1

```diff
@@ -0,0 +1,43 @@
+. "$PSScriptRoot/../end-to-end-tests-prelude.ps1"
+
+if ($IsLinux) {
+    $triplet = 'x64-linux-e2e'
+    $host_triplet = 'x64-linux'
+} else if ($IsMacOS) {
+    $triplet = 'x64-osx-e2e'
+    $host_triplet = 'x64-osx'
+} else if ($IsWindows) {
+    $triplet = 'x64-windows-e2e'
+    $host_triplet = 'x64-windows'
+} else {
+    Write-Warning "Unknown platform."
+    return
+}
+
+
+Write-Trace "test native qualified dependencies"
+
+$Overlays = "--overlay-ports=$PSScriptRoot/../e2e_ports/overlays", "--overlay-triplets=$PSScriptRoot/../e2e_ports/triplets"

```

Comment:
This is already part of the common args; we should just use those: https://github.com/microsoft/vcpkg-tool/blob/7f762bbaece17fe644e166b90e2cdacd429f55c9/azure-pipelines/end-to-end-tests-prelude.ps1#L15